snippets/javascript.json

```diff
@@ -0,0 +1,96 @@
+{
+  // Place your snippets for javascript here. Each snippet is defined under a snippet name and has a prefix, body and
+  // description. The prefix is what is used to trigger the snippet and the body will be expanded and inserted. Possible variables are:
+  // $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. Placeholders with the
+  // same ids are connected.
+  // Example:
+  // "Print to console": {
+  // 	"prefix": "log",
+  // 	"body": [
+  // 		"console.log('$1');",
+  // 		"$2"
+  // 	],
+  // 	"description": "Log output to console"
+  // }
```

Comment:
JSON does not support comments. I think this documentation as well as the one for JSReact would be best moved into `README.md`, after the existing XML and Python screenshots, with both this commented documentation and a screenshot of it in action in the VSCode editor.